test/featuregates_linter/cmd/feature_gates.go

```diff
@@ -168,9 +168,9 @@ func verifyOrUpdateFeatureList(rootPath, featureListFile string, update, version
 
 	if diff := cmp.Diff(featureList, baseFeatureList); diff != "" {
 		if versioned {
-			return fmt.Errorf("detected diff in versioned feature list (%s), diff: \n%s", versionedFeatureListFile, diff)
+			return fmt.Errorf("detected diff in versioned feature list (%s), please run hack/update-featuregates.sh to update list, diff: \n%s", versionedFeatureListFile, diff)
```

Comment:
seems like https://github.com/kubernetes/kubernetes/pull/127521  will make the tips here more accurate,remind me if I need to recover it.

Reply:
adding the DO NOT EDIT header is still useful as part of this PR imo.